### PR TITLE
Correct Metasploit Ansible role name

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Corrects the name of the Metasploit Ansible role
- Adds a system package dependency that is now required of a Python package dependency of CrackMapExec.

## 💭 Motivation and context ##

The Packer code does not build without these changes.

## 🧪 Testing ##

The AMI builds now; it did not before.  I also verified that I can run the Metsaploit and CrackMapExec tools on the new AMI.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.
- [x] Build and test a new staging AMI with these changes.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

**Comment by jsf9k** [comment]
@dav3r - I am out next week, and this PR probably won't get approved today, so I humbly request that you:

1. Finalize the version once this PR is approved.
2. Merge the PR.
3. Create a release from `develop`.

**Review by dav3r — APPROVED** [review]
LGTM! 👍

File: src/crackmapexec.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+# The CrackMapExec dependency impacket now requires the krb5-config
+# executable from the libkrb5-dev package.
+- name: Install ScareCrow dependencies
```
Comment (mcdonnnj) [inline src/crackmapexec.yml]:
```suggestion
- name: Install CrackMapExec dependencies
```

File: src/crackmapexec.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+# The CrackMapExec dependency impacket now requires the krb5-config
```
Comment (mcdonnnj) [inline src/crackmapexec.yml]: This is a [dependency for CrackMapExec](https://github.com/byt3bl33d3r/CrackMapExec/blob/593062ecfb4de28f21397862928b66f0eba5625f/requirements.txt#L28) and I'm not sure what changed but the [line that fails](https://github.com/pythongssapi/python-gssapi/blob/08e2b09fc68082fbec7b94aadb8c2c36efaea1b0/setup.py#L128) is not new.

```suggestion
# The CrackMapExec dependency gssapi requires the krb5-config
```

**Review by mcdonnnj — APPROVED** [review]
Thanks for sorting this out. I just have some documentation suggestions for your consideration.

**Comment by dav3r** [comment]
> @dav3r - I am out next week, and this PR probably won't get approved today, so I humbly request that you:
> 
> 1. Finalize the version once this PR is approved.
> 2. Merge the PR.
> 3. Create a release from `develop`.

The tasks above have been completed.  The current release is now [Perennial Ptarmigan](https://github.com/cisagov/kali-packer/releases/tag/v0.5.3).  

Also, after the new Production AMI was successfully created, I ran the `terraform-post-packer` code to allow it to be used by our assessment accounts.

**Comment by jsf9k** [comment]
> The tasks above have been completed. The current release is now [Perennial Ptarmigan](https://github.com/cisagov/kali-packer/releases/tag/v0.5.3).
> 
> Also, after the new Production AMI was successfully created, I ran the `terraform-post-packer` code to allow it to be used by our assessment accounts.

A hearty "Thank you!" to @dav3r for taking care of this in my absence.

**Pull request merged**